GitTfs/Core/GitTfsRemote.cs

```diff
@@ -485,8 +485,10 @@ private string ProcessChangeset(ITfsChangeset changeset, LogEntry log)
                 if (changeset.Summary.Workitems.Any())
                 {
                     var workItemIds = TranslateWorkItems(changeset.Summary.Workitems.Select(wi => wi.Id.ToString()));
+                    var workItemTitles = changeset.Summary.Workitems.Select(wi => wi.Title);
```

Comment:
Thank you for the suggestion. 
The modification has been done und pull request was updated.